specification/domainservices/resource-manager/readme.md

```diff
@@ -39,6 +39,14 @@ input-file:
 - Microsoft.AAD/2017-01-01/domainservices.json
 ```
 
+### Tag: package-2017-06
+
+These settings apply only when `--tag=package-2017-06` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-06'
+input-file:
+- Microsoft.AAD/2017-06-01/domainservices.json
```

Comment:
note that this path is *wrong*, the filename is `DomainServicesSwagger.json`. I've just fixed that for the previous API version, please make sure you also adjust this accordingly.